WordPress/Classes/ViewRelated/People/PersonViewController.swift

```diff
@@ -0,0 +1,154 @@
+import Foundation
+import UIKit
+import WordPressShared
+
+/// Displays a Blog's User Details
+///
+class PersonViewController : UITableViewController {
+    
+    // MARK: - Public Properties
+    var person  : Person!
+    var blog    : Blog!
+    
+    
+    // MARK: - View Lifecyle Methods
+    override func viewDidLoad() {
+        assert(person != nil)
+        assert(blog != nil)
+        
+        super.viewDidLoad()
+        
+        title = person.fullName.nonEmptyString() ?? NSLocalizedString("Blog's User", comment: "Blog's User Profile. Displayed when the name is empty!")
+        WPStyleGuide.configureColorsForView(view, andTableView: tableView)
+    }
+    
+    
+    // MARK: - UITableView Methods
+    override func tableView(tableView: UITableView, didSelectRowAtIndexPath indexPath: NSIndexPath) {
+        tableView.deselectSelectedRowWithAnimation(true)
+        
+        let handlerMap = [
+            roleCell    : #selector(handleRoleWasPressed),
+            removeCell  : #selector(handleRemoveWasPressed)
+        ]
+
+        guard let cell = tableView.cellForRowAtIndexPath(indexPath), let handler = handlerMap[cell] else {
+            return
+        }
+        
+        performSelector(handler)
+    }
+    
+    // MARK: - Storyboard Methods
+    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
+        guard let roleViewController = segue.destinationViewController as? RoleViewController else {
+            return
+        }
+        
+        roleViewController.role = person.role
+        roleViewController.onChange = { newRole in
+// TODO: JLP May.3.2016. Wire the Save OP
+        }
+    }
+    
+    
+    // MARK: - Action Handlers
+    @IBAction func handleRoleWasPressed() {
+        performSegueWithIdentifier(roleSegueIdentifier, sender: nil)
+    }
+    
+    @IBAction func handleRemoveWasPressed() {
+// TODO: JLP May.3.2016. Implement Me
```

Comment:
Again, this one is a bit easier to understand, but I'd suggest making it clearer when this is coming up, and any other relevant info.